src/cmd/create.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/godbus/dbus/v5"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-	"golang.org/x/crypto/ssh/terminal"
+	"golang.org/x/term"
```

Comment:
I am curious - what was the problem?  I never understood the different `term` and `terminal` APIs in Go.

[We were using](https://github.com/containers/toolbox/commit/950f510872404da8089ad7ce5911d8342bf7a116) `golang.org/x/crypto/ssh/terminal` because that's what Podman was using for the same purpose.

Also, don't we have to update `src/go.mod` for this?

Reply:
The `terminal` module has been deprecated in favor of the `term` module. The linter highlighted the fact.

`go.mod` already had a reference to the `term` module inside, so no change was needed.

Reply:
> The `terminal` module has been deprecated in favor of the `term` module.

I see.  Cool.

> `go.mod` already had a reference to the `term` module inside, so no
> change was needed.

The thing is that `go build` doesn't like the `// indirect` marker.

Submitted: https://github.com/containers/toolbox/pull/982

Reply:
Oh, missed that when applying the fix. Thanks!